From 69d9cb931d7ea172846f47724a6b15df11097c5f Mon Sep 17 00:00:00 2001
From: Xiao Chen <xiao@cloudera.com>
Date: Mon, 7 Mar 2016 17:11:19 -0800
Subject: [PATCH 1311/2848] CLOUDERA-BUILD. Revert "CLOUDERA-BUILD. Disable
 TestBalancer."

This reverts commit 10a81881bd711d8b0e98250a5c261bb2da8f5de6.
Re-enabling the test now that CDH-37778 is fixed.

Change-Id: I28d86b501aae6a91f45eb2f114777e0a6e41d77a
---
 cloudera/post_commit_hook.sh |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cloudera/post_commit_hook.sh b/cloudera/post_commit_hook.sh
index 57f991b..2ef7660 100755
--- a/cloudera/post_commit_hook.sh
+++ b/cloudera/post_commit_hook.sh
@@ -42,7 +42,7 @@ EOF
 
 # Invoke grind to run tests
 grind -c ${DIR}/supertest/grind.cfg config
-grind -c ${DIR}/supertest/grind.cfg test --artifacts -r 3 -e TestRM -e TestWorkPreservingRMRestart -e TestRMRestart -e TestContainerAllocation -e TestMRJobClient -e TestCapacityScheduler -e TestDelegatingInputFormat -e TestMRCJCFileInputFormat -e TestJobHistoryEventHandler -e TestCombineFileInputFormat -e TestAMRMRPCResponseId -e TestSystemMetricsPublisher -e TestNodesListManager -e TestRMContainerImpl -e TestApplicationMasterLauncher -e TestRMWebApp -e TestContainerManagerSecurity -e TestResourceManager -e TestParameterParser -e TestNativeCodeLoader -e TestRMContainerAllocator -e TestMRIntermediateDataEncryption -e TestWebApp -e TestCryptoStreamsWithOpensslAesCtrCryptoCodec -e TestDNS -e TestBalancer
+grind -c ${DIR}/supertest/grind.cfg test --artifacts -r 3 -e TestRM -e TestWorkPreservingRMRestart -e TestRMRestart -e TestContainerAllocation -e TestMRJobClient -e TestCapacityScheduler -e TestDelegatingInputFormat -e TestMRCJCFileInputFormat -e TestJobHistoryEventHandler -e TestCombineFileInputFormat -e TestAMRMRPCResponseId -e TestSystemMetricsPublisher -e TestNodesListManager -e TestRMContainerImpl -e TestApplicationMasterLauncher -e TestRMWebApp -e TestContainerManagerSecurity -e TestResourceManager -e TestParameterParser -e TestNativeCodeLoader -e TestRMContainerAllocator -e TestMRIntermediateDataEncryption -e TestWebApp -e TestCryptoStreamsWithOpensslAesCtrCryptoCodec -e TestDNS
 # TestDNS fails only on supertest. CDH-37451
 
 # Cleanup the grind folder
-- 
1.7.9.5

